library/entropy_poll.c

```diff
@@ -18,15 +18,9 @@
  */
```

Comment:
I still have a doubt about the commit message, shouldn't it be "glibc" in place of "libc"?

Reply:
The context is “Using glibc getrandom()”, but ok, I'll clarify.

Reply:
I rewrote that commit message.